src/awfy.js

```diff
@@ -515,10 +515,15 @@ export const CONFIG = {
       categories: DESKTOP_CATEGORIES,
     },
     mac: {
```

Comment:
This will change the URL for the dashboard, so I'd prefer "mac" to just point to the most relevant version. I can look into adding Big Sur. Does this have any Apple Silicon specific identifier? Not all macs running Big Sur are Apple Silicon.

Reply:
@jmaher I've pushed an update that adds Big Sur (M1)